tests/Services/ConvertToJsonServiceTest.php

```diff
@@ -17,11 +17,6 @@ public function testConvert_success()
         $this->assertSame(200, http_response_code());
     }
 
-    public function testConvert_success_unexpected_error()
```

Comment:
why has this been removed?

Reply:
We deleted that because it seemed to us that it doesn't do anything, we thought that it is an integration test not a unit test,and deleted it, but I can put that back if we need to .

Reply:
it would be an integration test, but the mark as skipped is there to tell other developers that, otherwise it just looks like you forgot to test it